ci/cloudbuild/notifiers/bigquery/deploy.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# There are a few commmands that need to be run in order to deploy a change to
+# the bigquery.yaml file, and this script runs those commands.
+
+set -euo pipefail
+
+source "$(dirname "$0")/../../../lib/init.sh"
+source module ci/lib/io.sh
+cd "${PROGRAM_DIR}"
+
+readonly BUCKET="gs://cloud-cpp-testing-resources-configs/google-cloud-cpp"
+io::log_h1 "Copying bigquery.yaml to ${BUCKET}"
+gsutil cp bigquery.yaml gs://cloud-cpp-testing-resources-configs/google-cloud-cpp
+
+readonly SERVICE="google-cloud-cpp-gcb-bigquery-notifier"
+io::log_h1 "Deploying Cloud Run service ${SERVICE}"
+run_args=(
+  "--image=us-east1-docker.pkg.dev/gcb-release/cloud-build-notifiers/bigquery:latest"
+  "--no-allow-unauthenticated"
+  "--update-env-vars=CONFIG_PATH=${BUCKET}/bigquery.yaml,PROJECT_ID=936212892354"
+  "--project=cloud-cpp-testing-resources"
+  "--platform=managed"
+  "--region=us-central1"
+)
+gcloud run deploy "${SERVICE}" "${run_args[@]}"
```

Comment:
FYI: this assumes Cloud Run is enabled.  If we were moving this to a different project it might fail until somebody enables the Cloud Run API.  No action required, I think you get a warning (and I have seen `gcloud` automatically enable APIs, but I have also seen it fail 🤷 )

Reply:
Good point. Actually, there are about 10 more steps that are needed for a first-time setup if we ever move to a new cloud project. I added a comment and link about this at the top of this script.